config/locales/en/errors.yml

```diff
@@ -50,3 +50,4 @@ en:
   staff_response:
     errors:
       must_reject_or_accept_visit: "A visit must either be accepted by choosing a date or at least one rejection reason"
+      lead_visitor_not_on_contact_list: The lead visitor is not on the contact list
```

Comment:
@StupidCodeFactory we should treat this error in the same way as any other rejection error. We don't need to have a specific error message - the visit will simply be rejected when the staff click 'Process'
Is that right do you think @chrishanes ?